docs/source/shaders/material/as_toon.rst

```diff
@@ -350,19 +350,19 @@ Contour Parameters
     Generate the contours based on the object ID.
 
 *Contour Material*
-    Generate contours based on the material ID. Objects with the same material ID would share contours.
+    Generate contours based on the material ID. Objects with the same material ID will share contours.
 
 *Contour Occlusion*
-    Generate contours based on ambient occlusion.
+    Generate contours based on ambient-occlusion.
```

Comment:
It was spelled correctly before: ambient occlusion.

Reply:
Occlusion does not use ambient occlusion. It uses depth differences between nearby points.